packages/app/src/domain/vaccine/milestones-view.tsx

```diff
@@ -58,68 +60,79 @@ export function MilestonesView(props: MilestoneViewProps) {
         position="relative"
         css={css({ listStyleType: 'none' })}
       >
-        <ListItemFirst isExpanded={isExpanded}>
+        <ListItemFirst>
           <CircleSmall />
           <Text fontWeight="bold" m={0} pl={`calc(1rem)`}>
             2021
           </Text>
         </ListItemFirst>
 
-        {milestones.length > MAX_ITEMS_VISIBLE && !isExpanded && (
+        {milestones.length > MAX_ITEMS_VISIBLE && (
           <ListItem>
             <Box pl={`calc(1rem + ${CIRCLE_SIZE}px)`}>
-              <ExpandButton color="link" onClick={handleExpand}>
-                {siteText.milestones.toon_meer}
+              <ExpandButton
+                color="link"
+                onClick={() => setIsExpanded((x) => !x)}
```

Comment:
Why did you use a function instead of this? Just curious

```suggestion
                onClick={() => setIsExpanded(!isExpanded)}
```

Reply:
I copy/pasted a code suggestion by @VWSCoronaDashboard10

Reply:
I think it's a good pattern to use a function when we're setting state based on the previous state.

Reply:
Cool, I'll change it back hehe

Reply:
haha sure, though in this case it didn't matter much but I think in general it's a nice pattern though. Especially within effects, it defeats the need of an extra parameter in the dependency array.

Reply:
Yeah I remember there were advantages to using a function, but I was wondering what it was in this case. Because I think using the variable name reads a little clearer, because you don't immediately know what x is in this case. But I'm fine with either.